### PR TITLE
[INLONG-2934][Manager] Check whether the collection is NULL to avoid NPE

### LINKED ISSUE #2934: [Bug][Manager] Manager client occured NPE since not check NULL
### What happened

failed when null

### What you expected to happen

pass when null

### How to reproduce

fault tolerance

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kuansix**
### Title Name: [INLONG-2934][Manager] Check whether the collection is NULL to avoid NPE

Fixes #2934

### Motivation

Fix NPE problem in manager-client.

### Verifying this change

- [X] Make sure that the change passes the CI checks.

### CODE REVIEW

**Comment by kuansix** [comment]
来信已收到，谢谢！

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java [hunk 1]
```diff
@@ -190,18 +190,24 @@ private InlongGroupContext generateSnapshot() {
         String inlongGroupId = currentGroupInfo.getInlongGroupId();
         //Fetch stream in group
         List<InlongStream> dataStreams = fetchDataStreams(inlongGroupId);
-        dataStreams.stream().forEach(dataStream -> groupContext.setStream(dataStream));
+        if (null != dataStreams) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java:193]: Please use `CollectionUtils.isNotEmpty()` to replace `null != xx`.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java [hunk 2]
```diff
@@ -190,18 +190,24 @@ private InlongGroupContext generateSnapshot() {
...
+        Map<String, String> errMsgs = logViews.stream()
+                .filter(x -> null != x.getEvent() && null != x.getException())
+                .collect(Collectors.toMap(EventLogView::getEvent, EventLogView::getException));
         inlongGroupContext.setErrMsg(errMsgs);
         return inlongGroupContext;
     }
 
     private List<InlongStream> fetchDataStreams(String groupId) {
         List<FullStreamResponse> streamResponses = managerClient.listStreamInfo(groupId);
+        if (null == streamResponses) {
```
Comment (healchow) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java:208]: ditto.

**Comment by healchow** [comment]
Fixed by https://github.com/apache/incubator-inlong/pull/2950

**Pull request closed without merging**